lib/Handler.js

```diff
@@ -173,6 +187,32 @@ class Handler {
   respond(error, response) {
     this.callback(error, response);
   }
+
+  /**
+   * Start profiling.
+   */
+  startProfiling() {
+    // We only require the profiler if we need it to ensure as little impact as
+    // possible when profiling is disabled.
+    if (typeof this.constructor.profiler === 'undefined') {
+      this.constructor.profiler = require('v8-profiler-lambda'); // eslint-disable-line global-require
+    }
+    if (typeof this.constructor.deflateSync === 'undefined') {
```

Comment:
Is it necessary to be this explicit as opposed to `if (this.constructor.deflateSync) {`?